frontend/src/libs/helpers/get-format-time.helper.ts

```diff
@@ -0,0 +1,22 @@
+const SECONDS_IN_MINUTE = 60;
+const TARGET_STRING_LENGTH = 2;
+
+const getFormatTime = (time: number): string => {
```

Comment:
```suggestion
const getFormattedTime = (time: number): string => {
```
Don't forget about the file name